### ISSUE TITLE
KeyImportError: Missing PEM header of type CERTIFICATE

### ISSUE THREAD

**Issue by 4l1fe** [ronf/asyncssh#109]
Pls give a hint what i do wrong when try connecting.

I use such call `asyncssh.connect(host, username=user, client_keys=['/home/user/.ssh/priv_key'])`
and get an error is in the title.

Type of a key is rsa is generated by `ssh-keygen -f priv_key`. Ssh version  `OpenSSH_7.2p2  OpenSSL 1.0.2g  1 Mar 2016`

**Comment by ronf** [comment]
By any chance do you have an extra blank line after the "-----END RSA PRIVATE KEY-----" line in your key file? It looks like the new X.509 certificate support I added in 1.11.0 might have made AsyncSSH sensitive to additional lines showing up after the private key. I may need to have it ignore anything which isn't a certificate in that case.

**Comment by 4l1fe** [comment]
yes, there is a blank line `cat -A ~/.ssh/priv_key`

```
-----END RSA PRIVATE KEY-----$
$
```

**Comment by ronf** [comment]
Agreed.

Here's a simple patch for this issue which you can apply to asyncssh/public_key.py:

```patch
index 424d26c..08284d6 100644
--- a/asyncssh/public_key.py
+++ b/asyncssh/public_key.py
@@ -2389,7 +2389,7 @@ def import_private_key_and_certs(data, passphrase=None):
         key, end = _decode_pem_private(lines, passphrase)
 
         lines = lines[end:]
-        certs = _decode_pem_certificate_list(lines) if lines else None
+        certs = _decode_pem_certificate_list(lines) if any(lines) else None
     else:
         key, end = _decode_der_private(data, passphrase)
 
```

This allows it to ignore extraneous blank lines, but it will still expect PEM certificate data if there's any non-blank lines in the file after the initial private key. There really shouldn't be other content mixed into these files.

I'll get this into the develop branch shortly, and plan on rolling it into the next release.

**Comment by ronf** [comment]
This fix is now released in AsyncSSH 1.11.1.
